docs/release-notes/release-notes-0.15.1.md

```diff
@@ -34,7 +34,13 @@
 
 * [Add minor comment](https://github.com/lightningnetwork/lnd/pull/6559) on
   subscribe/cancel/lookup invoice parameter encoding.
+
 * [Log pubkey for peer related messages](https://github.com/lightningnetwork/lnd/pull/6588).
+
+## Neutrino
+
+* Add [getblockhash command](https://github.com/lightningnetwork/lnd/pull/6510) to
```

Comment:
nit: consider extending the description to also take note of the new `lncli` command too.